packages/app/obojobo-document-engine/src/scripts/viewer/components/recommendation-component.js

```diff
@@ -0,0 +1,188 @@
+import './recommendation-component.scss'
+import React from 'react'
+import NavUtil from '../util/nav-util'
+import RecommendationUtil from '../util/recommendation-util'
+import GaugeChart from 'react-gauge-chart'
+
+const determineColor = (percent, settings) => {
+	for (const setting in settings) {
+		if (percent <= settings[setting].cutoff)
+			return settings[setting].color
+	}
+}
+
```

Comment:
Recommend moving each of these three components below (Summary, Recommendation, Recommendations) into separate files